ICSharpCode.Decompiler/TypeSystem/DecompilerTypeSystem.cs

```diff
@@ -104,6 +104,10 @@ public enum TypeSystemOptions
 		/// </summary>
 		NullabilityAnnotations = 0x400,
 		/// <summary>
+		/// If this option is active, 
```

Comment:
Incomplete comment

Reply:
I think, we can remove this setting completely, because local functions are not represented in the TS directly. `LocalFunctionMethod` is not created by the TS, but by `LocalFunctionDecompiler`.